glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: 283f982821510d6c8c5e48054919974b2b2b9664b838ec9f9a06705e19ed3d99
```

Comment:
I'm guessing it's still not updating properly?  It would be nice if we could get a real glide update done: maybe someone else knows what's going on?